src/Aspire.Hosting/DistributedApplication.cs

```diff
@@ -95,16 +279,60 @@ public virtual async Task StopAsync(CancellationToken cancellationToken = defaul
         await _host.StopAsync(cancellationToken).ConfigureAwait(false);
     }
 
-    /// <inheritdoc cref="HostingAbstractionsHostExtensions.RunAsync" />
+    /// <summary>
+    /// Runs an application and returns a Task that only completes when the token is triggered or shutdown is
+    /// triggered and all <see cref="IHostedService" /> instances are stopped.
+    /// </summary>
+    /// <param name="cancellationToken">The token to trigger shutdown.</param>
+    /// <returns>The <see cref="Task"/> that represents the asynchronous operation.</returns>
+    /// <remarks>
+    /// <para>
+    /// When the .NET Aspire AppHost is launched via <see cref="DistributedApplication.RunAsync"/> there are
+    /// two possible modes that it is running in:
+    /// </para>
+    /// <list type="number">
+    /// <item>Run mode; in run mode the AppHost runs until a shutdown of the app is triggered
+    /// either by the users pressing CTRL-C, the debugger detaching or the browser associated
+    /// with the dashboard being closed.</item>
+    /// <item>Publish mode; in publish mode the AppHost runs just long enough to generate a
```

Comment:
```suggestion
    /// <item>Publish mode; in publish mode the application host runs just long enough to generate a
```

Reply:
We've been calling it just "app host" in docs.